src/plugins/discover/public/application/main/components/sidebar/discover_sidebar_responsive.tsx

```diff
@@ -275,7 +284,7 @@ export function DiscoverSidebarResponsive(props: DiscoverSidebarResponsiveProps)
           <EuiButton
             contentProps={{
               className: 'dscSidebar__mobileButton',
-              id: DISCOVER_TOUR_STEP_ANCHOR_IDS.addFields,
+              // id: DISCOVER_TOUR_STEP_ANCHOR_IDS.addFields, // TODO: fix tour step for mobiel view
```

Comment:
One potential way to fix it, wrap the inner content of the `EuiButton` with
`<span data-test-subj="fieldListGroupedAvailableFields">`

Reply:
much better solution, you've implemented @jughosta 👍